### PR TITLE
Lock jsonschema to 3.2.0

### PR DESCRIPTION
**Pull request by adammkelly**

### CODE REVIEW

**Comment by p1c2u** [comment]
why it needs to be locked to version 3.2?

File: requirements.txt [hunk 1]
```diff
@@ -1,4 +1,4 @@
 isodate
-jsonschema
+jsonschema==3.2.0
```
Comment (sthagen) [inline requirements.txt:2]: Would maybe `>=3.2.0` be an option or is development so fast that users do not need wait  long until a dependency is bumped? … and maybe sync the build data files also? I noticed that `setup.cfg` still has `jsonschema>=3.0.0`

**Comment by adammkelly** [comment]
We got around this by adding jsonschema==3.2.0 in our requirements. What we were seeing at the time was this package tried to install jsonschema 4.0.0 which got openapi-schema-validator into a recursive loop which lasted longer than our worker timeout of 60 seconds. It never got past the schema validation.

Similar problem here: https://github.com/p1c2u/openapi-spec-validator/pull/132

**Comment by sthagen** [comment]
Well the spec validator requires the schema validator (this repo/package) - why specific the jsonschema explicitly downstream, when it is delivered upstream already? ... pinning to me is always a future problem (synchronized re-pinning or unpinning).
As both repos are under the same roof, why not coordinate by removing jsonschema dep form the spec-validator and add in here schema-validator a greater or equal pinning of the jsonschema depending on the required capabilities? ... JSON Schema 2020-12 draft comes to my mind ...

**Comment by p1c2u** [comment]
jsonschema requirement will be update to fix deprecation warning hence closing this one.

**Pull request closed without merging**